impf/decorators.py

```diff
@@ -84,7 +84,10 @@ def func(self, *args, **kwargs):
                 self.keep_browser = settings.KEEP_BROWSER_CRASH
             else:
                 sleep(10)
-            if not self.keep_browser: self.driver.close()
```

Comment:
As seen on line 84, `self.keep_browser` is used for a variety of use-cases; not only reusing the browser.

Reply:
That's the reason, why I split the meaning of settings.KEEP_BROWSER (renamed to REUSE_BROWSER) and Browser.keep_browser.
keep_browser now indicates that the currently used browser should not be closed or reused to allow for manual inspection or interaction. This could be the case after a crash (with settings.KEEP_BROWSER_CRASH set to true) or when appointments are available.
REUSE_BROWSER instructs to reuse a browser, but only if it should not be kept (indicated by keep_browser).

In the current/old code, if a crash occurred, keep_browser would be set to true, but would be reused when KEEP_BROWSER was set to true, which would not be what the code tried to do, I guess.